README.md

```diff
@@ -75,6 +75,31 @@ Type: `object`
 
 The element attributes
 
+### #setPluginVariable(pluginName, variableName, variableValue)
```

Comment:
I believe it would make more sense to use `addPluginVariable` instead. Because it's possible to call it multiple times.